msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CompileAppManifestTaskBase.cs

```diff
@@ -73,13 +73,6 @@ public abstract class CompileAppManifestTaskBase : XamarinTask
 		public string TargetArchitectures { get; set; }
 		#endregion
 
-		#region Outputs
-
-		[Output]
-		public ITaskItem CompiledAppManifest { get; set; }
```

Comment:
This output property was creating an empty Info.plist file on Windows for the Inputs/Outpus MSBuild check. If we don't want to have output props on this task we can run the `Touch` task after `CompileAppManifest`.

In my opinion having an ITaskItem output prop is still the best option, that way we ensure every time this task is run the file will be created on Windows.

Reply:
@emaf I made it an [Output] property:

https://github.com/xamarin/xamarin-macios/blob/d02e405963d3ec367fba2a583a04537648dd7b1d/msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CompileAppManifestTaskBase.cs#L37-L39

would that work?

Reply:
Yes, that should work, but the type should be `ITaskItem` :)

Reply:
@emaf ok, changed!